tracer-core/src/main/java/com/alipay/common/tracer/core/appender/TracerLogRootDaemon.java

```diff
@@ -64,6 +64,11 @@
         if (StringUtils.isBlank(loggingRoot)) {
             loggingRoot = System.getProperty("logging.path");
         }
+        // adapter to springboot 2.4.x
+        if (StringUtils.isBlank(loggingRoot)) {
+            loggingRoot = System.getProperty("logging.file.path");
```

Comment:
I think `logging.file.path` is not just a system property, but also an SpringBoot property.

Reply:
> I think `logging.file.path` is not just a system property, but also an SpringBoot property.
yeah, But there is not a change in the tracer, 3.1.0 version adds to the sofa-common-tools, and then through the CommonLoggingConfigurations from SOFABoot tin received all enviroment properties

Reply:
OK